### PR TITLE
Fixed broken link to mission statement

### PR DESCRIPTION
**Pull request by edwardearle**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Review-Date: 2022-04-19
 ---
 
 
-Welcome to our playbook. This document aims help our engineering, product, and design teams to operate within a clear framework of practices and acheive [our mission]().
+Welcome to our playbook. This document aims help our engineering, product, and design teams to operate within a clear framework of practices and acheive [our mission](1.-Welcome\Mission).
```
Comment (91dave) [inline docs/README.md:11]: Do we not need a forward slash `/` for a link/url?

Comment (dingjimmy) [reply docs/README.md:11]: As this file is in the root directory, I'm pretty sure it will work either way.

**Review by dingjimmy — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (dingjimmy) [reply docs/README.md]: Fixed it.

Comment (edwardearle) [reply docs/README.md]: Thank you both!

**Review by 91dave — APPROVED** [review]
Links look much better.

**Comment by edwardearle** [comment]
Merging broken links

**Pull request merged**